Detectors/TRD/simulation/include/TRDSimulation/Digitizer.h

```diff
@@ -108,6 +108,7 @@ class Digitizer
   std::vector<MCLabel> mMergedLabels;                                            // temporary label container
   std::array<SignalContainer, constants::MAXCHAMBER> mSignalsMapCollection;      // container for caching signals over a timeframe
   std::deque<std::array<SignalContainer, constants::MAXCHAMBER>> mPileupSignals; // container for piled up signals
+  std::array<bool, constants::MAXCHAMBER> mFlagVdriftOutOfRange;                 // flag chambers in which the drift velocity is out of range for which a parameterization exists
```

Comment:
`std::array<bool, constants::MAXCHAMBER> mFlagVdriftOutOfRange{false};` to ensure the right initialization.